src/seedu/addressbook/data/tagging/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tagging;
+
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.*;
+
+public class Tagging {
+    public final String operationName;
```

Comment:
The better way would be to use an enum, as it only represents add or delete operations.